docs/user-guide.md

```diff
@@ -184,6 +193,22 @@ To replay a conversation:
 
 The replayed conversation is displayed as a new conversation with the `[Replay]` tag in the list of your conversations. 
 
+### Parameterized Replay
+
+If you want to make your own chats or stories with custom variables and share them with your team, you can use a feature called Parameterized Replay. Use it to set up a chat or a story that others can follow, but provide their own details or answers to create a personalized experience. This makes chat a much more useful tool for sharing information and collaboration.
+
+How it works: 
+
+Let's create a simple Travel Guide conversation, which will take a user though several stepts to clarify information and provide the answer.
+
+1. Start by [creating a special prompt](#create-1) with [variables](#variables). Use templates (this `{{VariableName|DefaultValue}}` or this `{{VariableName}}`) to add variables into your prompt with or without default values. For example: `I'd like to travel to {{country|Japan}}. Could you please suggest {{num-attractions|10}} of the best attractions? I will be there for {{num-days}}. Thank you.`
+2. To use it, type `/` in the conversation text input area and select your prompt. You will be prompted to provide your inputs for variables in a pop-up window. Enter your inputs and click `Submit`. The prompt is now populated with your inputs in the conversation text box. Click `Play` to start.
```

Comment:
Do you mean 'Send a message to a chat?' 'Start replay' button exists in Replay mode only.

Reply:
fixed